src/Root.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  Navigate,
+  Route,
+  HashRouter as Router,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage, NotFoundPage, PeoplePage } from './pages';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="home" element={<Navigate to={'/'} replace />} />
+
+          <Route path="people" element={<PeoplePage />}>
+            <Route path=":personSlug" element={<PeoplePage />} />
```

Comment:
The nested route for `PeoplePage` uses the same component for both the parent route and the nested route with `:personSlug`. Ensure that `PeoplePage` is designed to handle both cases, or consider using a different component for the nested route if they require different logic or data.